spec/models/agents/google_flights_spec.rb

```diff
@@ -47,6 +48,11 @@
       @checker.options['adultCount'] = nil
       expect(@checker).not_to be_valid
     end
+    
+    it "should require preferredCabin" do 
+      @checker.options['preferredCabin'] = nil
+      expect(@checker).not_to be_valid
```

Comment:
I think this will make existing Agents invalid.  To test this, try switching to the master branch, making an agent, then switching to your branch and seeing if you have any issues interacting with it's show page, etc. I think it'd be best to make this field optional for backwards compatibility, especially if the default behavior of the API is COACH anyway.